src/main/java/leto/tasklist/BadTaskIndexException.java

```diff
@@ -0,0 +1,10 @@
+package leto.tasklist;
+
+public class BadTaskIndexException extends InvalidTaskException {
+    public BadTaskIndexException(int CurrentIndex) {
+        super(
+            "Bad task index number, should be a integer more than 0 and less than: " +
+            (CurrentIndex+1) +
+            " see /list for all the tasks");
```

Comment:
Might be better to move `+` in front.
I noticed the same issue in several other places too

Reply:
👷  will fix!